CK.ActivityMonitor/Impl/ActivityMonitorExtension.cs

```diff
@@ -45,10 +44,10 @@ public static void MonitorEnd(this IActivityMonitor @this, string text = null, [
         /// <param name="fileName">Source file name of the emitter (automatically injected by C# compiler but can be explicitly set).</param>
         /// <param name="lineNumber">Line number in the source file (automatically injected by C# compiler but can be explicitly set).</param>
         /// <returns>True if the log should be emitted.</returns>
-        public static bool ShouldLogLine( this IActivityMonitor @this, LogLevel level, [CallerFilePath]string fileName = null, [CallerLineNumber]int lineNumber = 0 )
+        public static bool ShouldLogLine( this IActivityMonitor @this, LogLevel level, [CallerFilePath]string? fileName = null, [CallerLineNumber]int lineNumber = 0 )
         {
             var h = ActivityMonitor.SourceFilter.FilterSource;
-            int combined = h == null ? 0 : (int)h( ref fileName, ref lineNumber ).LineFilter;
+            int combined = h == null ? 0 : (int)h( ref fileName, ref lineNumber ).LineFilter; //null warn there.
```

Comment:
IIRC CallerFilePath may not fill the path, so fileName may be null there.
The filter is implemented with a ConcurrentDictionnary and will throw if we try to access null.